script/release-notes/index.js

```diff
@@ -1,473 +1,139 @@
+#!/usr/bin/env node
+
 const { GitProcess } = require('dugite')
-const Entities = require('html-entities').AllHtmlEntities
-const fetch = require('node-fetch')
-const fs = require('fs')
-const GitHub = require('github')
 const path = require('path')
 const semver = require('semver')
 
-const CACHE_DIR = path.resolve(__dirname, '.cache')
-// Fill this with tags to ignore if you are generating release notes for older
-// versions
-//
-// E.g. ['v3.0.0-beta.1'] to generate the release notes for 3.0.0-beta.1 :) from
-//      the current 3-0-x branch
-const EXCLUDE_TAGS = []
+const notes = require(path.resolve(__dirname, 'notes.js'))
```

Comment:
Mostly for my curiosity, how is this different from `require('./notes.js')`?

Reply:
...because I didn't realize you could do that. :grimacing: 

I agree, that's much simpler. Fixed e712a78cc